Tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -0,0 +1,126 @@
+<?php
+/**
+ * Go! AOP framework
+ *
+ * @copyright Copyright 2015, Lisachenko Alexander <lisachenko.it@gmail.com>
+ *
+ * This source file is subject to the license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Go\Symfony\GoAopBundle\Tests\DependencyInjection;
+
+use Go\Symfony\GoAopBundle\DependencyInjection\Configuration;
+use Go\Symfony\GoAopBundle\DependencyInjection\GoAopExtension;
+use Matthias\SymfonyDependencyInjectionTest\PhpUnit\AbstractExtensionConfigurationTestCase;
+
+/**
+ * Class ConfigurationTest
+ */
+class ConfigurationTest extends AbstractExtensionConfigurationTestCase
+{
+    /**
+     * @test
+     */
+    public function itHasReasonableDefaults()
+    {
+        $expectedConfiguration = [
+            'cache_warmer' => true,
+            'doctrine_support' => false,
+            'options' => [
+                'features' => 0,
+                'app_dir' => '%kernel.root_dir%/../src',
+                'cache_dir' => '%kernel.cache_dir%/aspect',
+                'debug' => '%kernel.debug%',
+                'include_paths' => [],
+                'exclude_paths' => []
+            ]
```

Comment:
Could you please, enable assignment and key-value alignment in IDE? Much easier to follow text on the screen when text is aligned 🤓 

Same is applied to following code blocks too.